src/components/Timesheet/index.tsx

```diff
@@ -64,21 +45,87 @@ const Timesheet = () => {
+        setTimeTracked(() => parseInt(e.target.value));
+    };
+
     return (
-        <table>
-            <thead>
-                <tr>
-                    <td>Monday</td>
-                    <td>Tuesday</td>
-                    <td>Wednesday</td>
-                    <td>Thursday</td>
-                    <td>Friday</td>
-                    <td>Saturday</td>
-                    <td>Sunday</td>
-                </tr>
-            </thead>
-            <tbody></tbody>
-        </table>
+        <div className={styles.timesheet}>
+            <table className={styles.table}>
+                <thead>
+                    <tr>
+                        {days.map((day) => (
+                            <td key={day}>
+                                <div className={styles.days}>{day}</div>
+                            </td>
+                        ))}
+                    </tr>
+                </thead>
+
+                <tbody>
+                    {dates.length > 0 &&
+                        dates.map((week) => (
+                            <tr
+                                key={JSON.stringify(week[0])}
+                                className={styles.row}
+                            >
+                                {week.map(
+                                    (each: {
+                                        date:
+                                            | string
+                                            | React.ReactChild
+                                            | React.ReactFragment
+                                            | React.ReactPortal
+                                            | null
+                                            | undefined;
+                                        month: string;
+                                    }) => (
+                                        <td
+                                            className={
+                                                each.month.toString() ==
+                                                calendar.month.toString()
+                                                    ? `${styles.cell} ${styles.current}`
+                                                    : `${styles.cell} ${styles['not-current']}`
+                                            }
+                                            key={JSON.stringify(each)}
+                                        >
+                                            <div
+                                                className={styles.cell_wrapper}
+                                            >
+                                                <div className={styles.date}>
+                                                    {each.date}/
+                                                    {Number(each.month) + 1}
+                                                </div>
+
+                                                <input
```

Comment:
Even though I love how simple it would be with inputs, we need it to contain more information (I added the design for logging time today), so please change this to some other non-input HTML element

Reply:
Replaced as suggested.